subgraphs/reviews/reviews.graphql

```diff
@@ -0,0 +1,36 @@
+schema @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@key", "@override", "@shareable"]) {
+  query: Query
+}
+
+type Product implements ProductItf @key(fields: "id") {
+  id: ID!
+  reviewsCount: Int!
+  reviewsScore: Float! @shareable @override(from: "products")
+  reviews: [Review!]!
+}
+
+interface ProductItf {
+  id: ID!
+  reviewsCount: Int!
+  reviewsScore: Float!
+  reviews: [Review!]!
+}
+
+type Query {
```

Comment:
@patrick91 @michael-watson can we drop the `_service` and `_entities` for the Strawberry subgraph - does it automatically add them to the schema at runtime like Apollo Server with `subgraph-js`?

Reply:
since `strawberry` is code first I think this schema file is provided just for reference

Reply:
> since `strawberry` is code first I think this schema file is provided just for reference

yes, that's correct :) but this is also the schema that you get when running `strawberry export-schema`. I think I'll make a new release of strawberry that has a `strawberry export-schema --federation` command that removes those